File: Alternativas/ej4.py
# -*- coding: utf-8 -*-

#Crea un programa que pida al usuario dos números y
#muestre su división si el segundo no es cero, o un 
#mensaje de aviso en caso contrario.

#1.almacear primer número (real)
#2.almacenar segundo número (real)
#3.si numero2 es distinto de 0
#4.divide el primero entre el segundo
#5.en caso contrario muestra mensaje de aviso

print("--->Dividir dos números<---")
numero1 = float(input("Introduce el dividendo> "))
numero2 = float(input("Introduce el divisor> "))
if(numero2==0):
	print("---------------------------------")
	print("  no se puede dividir entre 0")
	print("---------------------------------")
else:
	print(numero1," / ",numero2," = ",numero1/numero2)

File: Java/Alternativas/ej4.java
import java.util.Scanner; //Importar la clase Scanner 

/**
 * Crea un programa que pida al usuario dos números y 
 * muestre su división si el segundo no es cero, o un mensaje de aviso en caso contrario.
 * @author Alberto Miguel Martinez Jimenez
 */
/**
 * Algoritmo:
 * Almacenar numero1
 * Almacenar numero2
 * Si numero2 es 0 mostrar aviso
 * Si no mostrar numero1/numero2
 */
public class ej4 {

	public static void main(String[] args) {
		double numero1;
		double numero2;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("---Dividir---");
		System.out.println("Introduce dos números> ");
		numero1 = scanner.nextDouble();
		numero2 = scanner.nextDouble();
		
		if(numero2!=0) {
			System.out.println("El resultado de la división es "+numero1/numero2);
		}else{
			System.out.println("El segundo número no puede ser 0");
		}
	}

}
